discord/http.py

```diff
@@ -1342,8 +1342,23 @@ def create_from_template(self, code: str, name: str, region: str, icon: Optional
             payload["icon"] = icon
         return self.request(Route("POST", "/guilds/templates/{code}", code=code), json=payload)
 
-    def get_bans(self, guild_id: Snowflake) -> Response[List[guild.Ban]]:
-        return self.request(Route("GET", "/guilds/{guild_id}/bans", guild_id=guild_id))
+    def get_bans(
+            self,
+            guild_id: Snowflake,
+            limit: Optional[int] = None,
+            before: Optional[Snowflake] = None,
+            after: Optional[Snowflake] = None,
```

Comment:
```suggestion
        self,
        guild_id: Snowflake,
        limit: Optional[int] = None,
        before: Optional[Snowflake] = None,
        after: Optional[Snowflake] = None,
```

Reply:
[PEP8](https://peps.python.org/pep-0008/#indentation) states "Add 4 spaces (an extra level of indentation) to distinguish arguments from the rest."